frontend/packages/policy-editor/src/components/ExpandablePolicyCard/ExpandablePolicyCard.tsx

```diff
@@ -28,6 +28,16 @@ import {
 import { useTranslation } from 'react-i18next';
 import { StudioButton } from '@studio/components';
 
+const allowedActionsId: string[] = [
```

Comment:
Maybe we should rename this to `wellKnownActionsId`, since this is all about which actions we know about? 😊 
```suggestion
const wellKnownActionsId: string[] = [
```